AWGL/TSO500_post_processing
# Myeloid panel

## Change description
- Add bed files for myeloid, CLL and MPN panels
- Update `vendorCaptureBed_100pad_updated.bed` and `TSO_extra_padding_chr.interval_list` so that they both include these regions
- Added script to pull through selected variants that were previously filtered by the blacklist (and new environment called pysam to run it in, it clashed with the Python version in the `TSO500_post_processing` env)

## Justification for minor update
No effect on sensitivity

## Testing description
Verified in CC-22-193-GEN

Pipeline was run end to end in `/Output/validations/TSO500/myeloid_test_run/`
- Check that virtual panels were applied correctly and upload to the database
- Check that blacklisted variants were pulled through as expected
- Check VAF is within the validated rage from TSO500 in reference standards
- Check difference in transcripts

## Test data location
`/Output/validations/TSO500/myeloid_test_run/`

`S:\MedGen\SHARED\Lab Genetics\TECHNICAL SECTION\NGS & Sequencing\Developmental work\NovaSeq\Validation_Projects\Comprehensive Cancer Panel\CC-22-193 Myeloid`

## Verification results
Pipeline run through successfully and all tests passed. Results are included in the writeup for CC-22-193-GEN
